scss/civicrm/_mixins.scss

```diff
@@ -1,3 +1,11 @@
+/*
+* Use Open Sans as primary font.
```

Comment:
The word **primary** is not applicable here. If you apply this mixin to a specific `div` this does not make this font primary. Please describe it as something like "Force Open Sans font family". Always explain what the function/mixin do as precise as possible.

Reply:
will do! thanks